SPECS/kernel/CVE-2022-3108.nopatch

```diff
@@ -0,0 +1,3 @@
+CVE-2022-3108 - Fix present in 5.10.174
+Upstream: 8027a9ad9b3568c5eb49c968ad6c97f279d76730  
```

Comment:
This does not match the patch posted on NIST and looks to be for a different CVE

https://nvd.nist.gov/vuln/detail/CVE-2022-3108

Reply:
uh oh, will retrace my steps.

Reply:
Got it mixed up with a very similar commit: https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v5.10.176&id=25bc9fd4c8d13036c0656a00ebedc2cff4650a91